src/Npgsql/NpgsqlConnector.cs

```diff
@@ -1291,7 +1305,7 @@ internal IBackendMessage SkipUntil(BackendMessageCode stopAt)
 
             while (true)
             {
-                var msg = ReadMessage(false, DataRowLoadingMode.Skip).GetAwaiter().GetResult()!;
+                var msg = ReadMessageWithoutCancellation(false, DataRowLoadingMode.Skip).GetAwaiter().GetResult()!;
```

Comment:
There may be cases where we want to do SkipUntil with cancellation (after all it's just a wrapper over ReadMessage). But looking at the specific places where we use this, it probably doesn't matter.

Reply:
Well, there are:
- 2 copy disposes
- 2 derive parameters for query
- 1 perform keepalive

Reply:
Agreed